pennylane/_device.py

```diff
@@ -677,7 +677,8 @@ def expand_fn(self, circuit, max_expansion=10):
         if self.custom_expand_fn is not None:
             return self.custom_expand_fn(circuit, max_expansion=max_expansion)
 
-        return self.default_expand_fn(circuit, max_expansion=max_expansion)
+        new_tape = self.default_expand_fn(circuit, max_expansion=max_expansion)
+        return new_tape
```

Comment:
Is there an advantage to doing this before returning it? Or is this a debugging carry-over?

Reply:
debugging carry over now removed.